src/main/java/sqlline/BuiltInProperty.java

```diff
@@ -34,7 +34,7 @@
       new Application().getName2HighlightStyle().keySet()),
   COLOR("color", Type.BOOLEAN, false),
   CONFIRM("confirm", Type.BOOLEAN, false),
-  CONFIRM_PATTERN("confirmPattern", Type.STRING, DEFAULT),
+  CONFIRM_PATTERN("confirmPattern", Type.STRING, "^(?i:(DROP|DELETE))"),
```

Comment:
I think it would be better to announce this change clearly at the next release.

Reply:
Indeed it was originally mentioned in doc as `default-confirm-pattern: ^(?i:(DROP|DELETE))` but just `default` in code which was not `^(?i:(DROP|DELETE))`.
For that reason test behaves not fully correct
However may be it makes sense to announe

Reply:
@snuyanzin I confirm the manual and I understand this background. Thanks.